llvm/test/CodeGen/X86/apx/inline-asm-jr-constraint.ll

```diff
@@ -0,0 +1,25 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: not llc -mtriple=x86_64 < %s | FileCheck %s
+; RUN: not llc -mtriple=x86_64 -mattr=+egpr < %s | FileCheck %s
```

Comment:
You can use sth like
```
not llc -mtriple=x86_64 -mattr=+egpr < %s >%t1 2>%t2
FileCheck %s %t1
FileCheck %s %t2 --check-prefix=ERR
```

Reply:
I'll try. If so, we can refine llvm/test/CodeGen/X86/apx/asm-constraint.ll as well.

Reply:
[e752556](https://github.com/llvm/llvm-project/pull/92338/commits/e752556c06ac25d905c6e642bdcb5e9244db5da3)